src/current/v24.2/auto-tls.md

```diff
@@ -139,7 +139,7 @@ cockroach init --certs-dir=certs --host=localhost:26257
 
 - [`cockroach cert`]({% link {{ page.version.version }}/cockroach-cert.md %})
 - [`cockroach auth-session`]({% link {{ page.version.version }}/cockroach-auth-session.md %})
-- [Authenticate to CockroachDB Self-Hosted Clusters]({% link {{ page.version.version }}/authentication.md %})
+- [Authenticate to CockroachDB {{ site.data.products.core }} Clusters]({% link {{ page.version.version }}/authentication.md %})
```

Comment:
This makes the linked text inconsistent with the title-case page title, but I'm OK with it. Perhaps lowercase the `Clusters` so it looks less weird?

Reply:
Done!